### PR TITLE
[SettingsCard] Add MaxWidth to icon presenter

### PR DESCRIPTION
**Pull request by niels9001**
Fixes: #286

### CODE REVIEW

File: labs/SettingsControls/src/SettingsCard/SettingsCard.xaml [hunk 1]
```diff
@@ -303,6 +303,7 @@
 
                             <ContentPresenter x:Name="PART_HeaderIconPresenter"
                                               Grid.RowSpan="1"
+                                              MaxWidth="{StaticResource SettingsCardIconMaxWidth}"
```
Comment (michael-hawker) [inline labs/SettingsControls/src/SettingsCard/SettingsCard.xaml:306]: Do we need something similar for the `ActionIcon` as well?

Comment (michael-hawker) [reply labs/SettingsControls/src/SettingsCard/SettingsCard.xaml:306]: And do we need both Width and Height to ensure things remain square?

**Comment by michael-hawker** [comment]
@niels9001 think I pointed this out in our chat, figured would be good to add here:

https://github.com/CommunityToolkit/Labs-Windows/blob/1e00d46b9ec54198fa0d16846019ec71663ea9b0/labs/SettingsControls/src/SettingsExpander/SettingsExpander.xaml#L79

This needs to be two-way template bound so it'll react to changes and update the property on the parent.

```xml
IsExpanded="{Binding IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
```

**Comment by michael-hawker** [comment]
Don't forget to bump version too ;)

**Comment by michael-hawker** [comment]
Other notes, we should provide light style tweaks (like MinWidth) for the (Slider, TextBox, ComboBox, NumberBox, AutoSuggestBox, Button?, Date/TimePicker?) here:

https://github.com/CommunityToolkit/Labs-Windows/blob/1e00d46b9ec54198fa0d16846019ec71663ea9b0/labs/SettingsControls/src/SettingsCard/SettingsCard.xaml#L139-L146

Also, if you set `ContentAlignment` to Vertical the ColSpan should include the content area on the right to provide the full area (in combination with HorizontalContentAlignment to Stretch usually).

**Comment by niels9001** [comment]
@michael-hawker On the contentalignment bug in Vertical mode - it wasn't related to the ColumnSpan (because the 4th column was set to Auto). The HorizontalContentAlignment was ignored. This is now fixed and I think is the expected behavior?

For example, a SettingsCard with a red Grid (with MinWidth="12")
```xml
<SettingsCard HorizontalContentAlignment="Left"...>
```

image

```xml
<SettingsCard HorizontalContentAlignment="Stretch"...>
```
image

``xml
<SettingsCard HorizontalContentAlignment="Right"...>
```
<img width="469" alt="image" src="https://user-images.githubusercontent.com/9866362/196671438-dc70dc5e-8e1b-4c51-b620-e837a0929a63.png">

File: labs/SettingsControls/samples/SettingsControls.Samples/SettingsCardSample.xaml [hunk 2]
```diff
@@ -1,21 +1,24 @@
-﻿<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
-<Page x:Class="SettingsControlsExperiment.Samples.SettingsCardSample"
```
Comment (michael-hawker) [inline labs/SettingsControls/samples/SettingsControls.Samples/SettingsCardSample.xaml:2]: Add an example for the indent?

File: labs/SettingsControls/src/SettingsCard/SettingsCard.xaml [hunk 3]
```diff
@@ -168,27 +132,115 @@
...
                             </Grid.RowDefinitions>
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="Auto" MinWidth="{ThemeResource SettingsCardLeftIndention}" />
                                 <ColumnDefinition Width="*" />
                                 <ColumnDefinition Width="Auto" />
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
+                            <ContentPresenter
```
Comment (michael-hawker) [inline labs/SettingsControls/src/SettingsCard/SettingsCard.xaml:141]: I think your local XAML Styler settings got flipped, there shouldn't be as many changes here? Try running the script on the commandline as the CI is failing?

**Review by michael-hawker — APPROVED** [review]
Sweet! Pointed out some clean-up for the project file, otherwise looks good. Fix that, get a clean build, and I'll merge and re-test the integration scenario I've been using. 🙂

**Pull request merged**